lib/components/PoolTypeSelector.jsx

```diff
@@ -43,36 +46,40 @@ const PrizePoolInputs = props => {
 
 const CompoundPrizePoolInputs = props => {
   return (
-    <>
-      <label
-        htmlFor={'token'}
-        className='mt-0 trans text-purple-300 hover:text-white'
+    <InputCard>
+      <InputLabel
+        title="Deposit Token"
+        description=""
       >
-        Yield service token to use:
-      </label>
-      <TokenDropdown onChange={props.handleTickerChange} />
-    </>
+        <TokenDropdown onChange={props.setCToken} />
+      </InputLabel>
+    </InputCard>
   )
 }
 
 const StakingPrizePoolInputs = props => {
   return (
-    <>
-      <TextInputGroup
-        id='_stakedTokenAddress'
-        label={
-          <>
-            Token Address:{' '}
-            <span className='text-default italic'>
-              (eg. '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984')
-            </span>
-          </>
-        }
-        placeholder='(eg. 0x1f9840a85d5af5bf1d1762f925bdaddc4201f984)'
-        required
-        onChange={e => props.setStakedTokenAddress(e.target.value)}
-        value={props.stakedTokenAddress}
-      />
-    </>
+    <InputCard>
+      <InputLabel
+        title="Staked Token Address"
+        description=""
```

Comment:
Could probably leave these lines out and handle the description `undefined` state in the `InputLabel` component.